go/kbnm/findbin.go

```diff
@@ -14,24 +15,24 @@ var errKeybaseNotFound = errors.New("failed to find the keybase binary")
 // findKeybaseBinary returns the path to a Keybase binary, if it finds it.
 func findKeybaseBinary(name string) (string, error) {
 	// Is it near the kbnm binary?
-	dir, err := utils.BinPath()
+	binPath, err := utils.BinPath()
 	if err == nil {
-		path := filepath.Join(dir, name)
-		if _, err := os.Stat(path); !os.IsNotExist(err) {
-			return path, nil
+		binPath := filepath.Join(path.Dir(binPath), name)
```

Comment:
Oh also, I think this is supposed to be `filepath.Dir`

Reply:
😱 oops yes i'll fix